core/src/main/java/hudson/Functions.java

```diff
@@ -1874,7 +1874,7 @@ public static String toEmailSafeString(String projectName) {
     /**
      * Obtains the host name of the Hudson server that clients can use to talk back to.
      * <p>
-     * This is primarily used in {@code slave-agent.jnlp.jelly} to specify the destination
+     * This is primarily used in {@code jenkins-agent.jnlp.jelly} to specify the destination
```

Comment:
https://github.com/search?q=org%3Ajenkinsci+%22slave-agent.jnlp%22&type=code looks like we could get away with just renaming the existing file instead of duplicating it.

If we need to keep the old file for some reason, would just have that `st:include` the new one if possible to cut down duplication.

Reply:
Thanks for catching that. It was supposed to be removed but while I was getting things working it crept back in. Removing ....